server/services/supportedPrisonsService.ts

```diff
@@ -0,0 +1,14 @@
+import { HmppsAuthClient, OrchestrationApiClient, RestClientBuilder } from '../data'
+
+export default class SupportedPrisonsService {
+  constructor(
+    private readonly orchestrationApiClientFactory: RestClientBuilder<OrchestrationApiClient>,
+    private readonly hmppsAuthClient: HmppsAuthClient,
+  ) {}
+
+  async getSupportedPrisonIds(): Promise<string[]> {
+    const token = await this.hmppsAuthClient.getSystemClientToken()
```

Comment:
In our other apps, we pass `user` to `getSystemClientToken()`, which gets incorporated into the system client token. Works without it though; value of `%ANONYMOUS%` is substituted.